llvm/utils/git/github-automation.py

```diff
@@ -567,9 +567,15 @@ def create_pull_request(self, owner: str, repo_name: str, branch: str) -> bool:
             print("PR already exists...")
             return True
         try:
+            commit_message = repo.get_commit(commits[-1]).commit.message
```

Comment:
The idea here is that usually the first commits are tests and the last one the functional patch?

Reply:
tests or some other prerequisite that's needed for the fix.  It's not always this way, but I'm not sure what else to put in the PR description.

Reply:
Let's say we have 3 commits. They can be either of `test-cleanup test-cleanup functional` or `functional test-fixup test-fixup` or `test-cleanup functional test-fixup`.

A heuristic may be to pick the commit with the longest commit message...